src/components/ControlBar.js

```diff
@@ -14,11 +14,17 @@ const controlBarStyle = {
   justifyContent: 'space-evenly',
 };
 
-const ConnectedControlBar = (props): React.MixedElement => {
+const ControlBar = (): React.MixedElement => {
+  const modalState = useSelector(state => state.displayModal);
```

Comment:
if this is a boolean our team uses the syntax isModalShown/isModalDisplayed etc. for booleans